hmpps-prisoner-search-indexer/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/resource/CompareIndexResource.kt

```diff
@@ -24,12 +24,11 @@ import uk.gov.justice.digital.hmpps.prisonersearch.indexer.services.CompareIndex
 @Tag(name = "OpenSearch index comparison")
 class CompareIndexResource(private val compareIndexService: CompareIndexService) {
   @GetMapping("/size")
-  @Tag(
-    name = "Simple OpenSearch index size comparison.",
-    description = """
```

Comment:
A newline at the beginning of the description appears to change the documentation to all be in `code` format, so bright purple 🤷